core/server/src/main/java/alluxio/worker/file/UnderFileSystemManager.java

```diff
@@ -54,35 +54,35 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
   // Input stream agent session index
-  private final IndexedSet.FieldIndex<InputStreamAgent> mInputStreamAgentSessionIdIndex =
-      new IndexedSet.FieldIndex<InputStreamAgent>() {
+  private final IndexedSet.UniqueFieldIndex<InputStreamAgent> mInputStreamAgentSessionIdIndex =
```

Comment:
Yes, It is. I originally let them be unique ones to test whether the unique and non-unique indexes can be distinguished with each other.